### PR TITLE
Update the hie-bios commit SHA

### PR DESCRIPTION
**Pull request by shayne-fletcher-da**
Might not have been necessary in the end to get hie-core working in emacs but as @cocreature says, there have been many fixes so this is probably a good idea anyway.

### CODE REVIEW

File: WORKSPACE [hunk 1]
```diff
@@ -548,7 +548,7 @@ hazel_repositories(
             hazel_github_external(
                 "mpickering",
                 "hie-bios",
-                "9f9fe00591c429c410475349560252ca7e622f1b",
+                "8427e424a83c2f3d60bdd26c02478c00d2189a73",
                 "0f9cf9ed897ed1faf1a36dadc06ba9bb3c525f8553da30bee0615294f4d00b00",
```
Comment (cocreature) [inline WORKSPACE]: You also need to update the hash.

**Comment by shayne-fletcher-da** [comment]
@moritzkiefer-da looks like this induces some required source code changes. Let's pick it up Monday.

**Comment by shayne-fletcher-da** [comment]
The last commit adds some tutorial documentation on Emacs integration.

File: WORKSPACE (hunk 1 above)
Comment (shayne-fletcher-da) [reply WORKSPACE]: ✔️

**Comment by cocreature** [comment]
@shayne-fletcher-da I’ve pushed a fix for the errors you got.

File: compiler/hie-core/hie-core-in-emacs.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
+# How to get `hie-core` working in Emacs
```
Comment (ndmitchell) [inline compiler/hie-core/hie-core-in-emacs.md]: I'd just fling this as a section in the normal readme for hie-core. I wouldn't have thought of hunting for an emacs specific one.

Comment (shayne-fletcher-da) [reply compiler/hie-core/hie-core-in-emacs.md]: Copy that @neil-da. I'll take care of it Monday. Initial doc. was just to get something going. I'll make it nice and integrate.

**Comment by shayne-fletcher-da** [comment]
> @shayne-fletcher-da I’ve pushed a fix for the errors you got.

@moritzkiefer-da , 🎸 !

**Pull request merged**